qiskit_experiments/randomized_benchmarking/rb_experiment.py

```diff
@@ -115,6 +117,10 @@ def _sample_circuits(
             circuits += self._generate_circuit(elements, element_lengths)
         return circuits
 
+    def set_epg_1_qubit(self, epg_1_qubit):
```

Comment:
I think dedicated option is not necessary. User can find this option if we provide kind docstring for set methods (this will be done in #116). Actually this method is inherited by the `InterleavedRBExperiment`, which isn't necessary to set this information (and shown in API doc of it).